v10.yaml

```diff
@@ -0,0 +1,389 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: node-feature-discovery
+---
```

Comment:
What on earth is this file? 🚀  😆

Reply:
hahahaha ups , `git add .` got me deleting.....